api/src/wfl/service/postgres.clj

```diff
@@ -78,7 +78,7 @@
 (defn update-workload-status!
   "Use `tx` to mark `workload` finished when all `workflows` are finished."
   [tx {:keys [id items] :as _workload}]
-  (let [query (format "SELECT id FROM %%s WHERE status NOT IN %s"
+  (let [query (format "SELECT id FROM %%s WHERE status IS NULL OR status NOT IN %s"
```

Comment:
While this fixes it, I'm not convinced execution should ever reach here if the workload hasn't been started. Maybe add a check at the top of the `update-workload!` implementation?

Reply:
Good idea, left this change since it is a "more correct" SQL query but added your suggestion